src/iotjs_binding.c

```diff
@@ -85,6 +86,43 @@ double iotjs_jval_as_number(jerry_value_t jval) {
 }
 
 
+bool iotjs_jbuffer_get_bufferwrap_or_null(jerry_value_t jval,
```

Comment:
The function name does not reflect to its behaviour. Please give a proper name to it.

Reply:
What do you think about the `iotjs_jbuffer_is_bufferwrap_or_string`?

Reply:
IMHO, `iotjs_jbuffer_get_bufferwrap` is enough